source/mesh_deformation/interface.cc

```diff
@@ -581,6 +603,67 @@ namespace aspect
 
 
 
+    template <int dim>
+    void MeshDeformationHandler<dim>::set_initial_topography()
```

Comment:
This function might become obsolete with @gassmoeller recent patch. We'll have to figure out who gets to merge first :-)

Reply:
Yes, that will be great! There are also some places in core.cc that will have to be reverted back then.